docker-compose.yml

```diff
@@ -160,6 +160,7 @@ services:
         JIRA_TASK_COLLECTOR_ENABLED: ${JIRA_TASK_COLLECTOR_ENABLED}
         JIRA_TRACKER_COLLECTOR_ENABLED: ${JIRA_TRACKER_COLLECTOR_ENABLED}
         JIRA_METADATA_COLLECTOR_ENABLED: ${JIRA_METADATA_COLLECTOR_ENABLED}
+        CVEORG_COLLECTOR_ENABLED: ${CVEORG_COLLECTOR_ENABLED}
```

Comment:
Note: should be added to ops repo (both spec files and local env files) as well.

Reply:
Thanks for the reminder, MR for this already exists.